website/docs/reference/web/portals-plugin.md

```diff
@@ -2,3 +2,160 @@
 title: Portals Plugin
 sidebar_label: Portals Plugin
 ---
+
+The PortalsPlugin class is the main way to interface with a Portal instance. It uses a [PubSub pattern](https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern) to publish data to a given topic and react accordingly.
```

Comment:
This makes it feel like the pub/sub is the main functionality of the plugin. Maybe:

The PortalsPlugin class is the main way to interface with a Portal instance. It has methods to easily pass messages back and forth to the native side via a publish/subscribe interface, and ways to pass data to a web view before it initializes.